README.md

```diff
@@ -51,6 +51,11 @@ console.log(isPoll(validPoll)) // => true
 - `parsePoll`,
 - `parsePosition`,
 
+### Parser Errors (useful for debugging)
+
+- `parsePollErrors`,
+- `parsePositionErrors`,
```

Comment:
can you give me some sample return values e.g. 

```js
parsePollErrors(poll)
=> {
  errors: {
    v1:  // .. missing a title!
    v2:  // .. missing new attribute added in v2! 
  }
}
```

Reply:
I think calling it `getPollErrors` might be more descriptive and less confusing. feels active like yeah I am really just wanting to get all those errors plz

Reply:
Nice, just changed the naming to `getPollErrors` as you suggest. Way better.